.gitignore

```
@@ -6,6 +6,12 @@ __pycache__/
 # C extensions
 *.so
 
+# Static and Media contents
+media/*
+!media/.gitkeep
+static/*
+!static/.gitkeep
```

similar things look better together.

```suggestion
media/*
static/*
!media/.gitkeep
!static/.gitkeep
```